tests/components/roku/test_remote.py

```diff
@@ -65,3 +67,25 @@ async def test_main_services(
     )
     assert mock_roku.remote.call_count == 3
     mock_roku.remote.assert_called_with("home")
+
+    mock_roku.remote.reset_mock()
+    await hass.services.async_call(
+        REMOTE_DOMAIN,
+        SERVICE_SEND_COMMAND,
+        {
+            ATTR_ENTITY_ID: MAIN_ENTITY_ID,
+            ATTR_COMMAND: ["left", "right"],
+            ATTR_NUM_REPEATS: 2,
+            ATTR_DELAY_SECS: 1,
+        },
+        blocking=True,
+    )
+    assert mock_roku.remote.call_count == 4
```

Comment:
Shouldn't this be 5 and it was 3 before this test and two repeats were specified?